stan/math/prim/scal/prob/gumbel_cdf.hpp

```diff
@@ -23,8 +23,8 @@ namespace stan {
   namespace math {
 
     /**
-     * Returns the Gumbel distribution cumulative distribution for the given 
```

Comment:
Any idea why these weren't being picked up by cpplint?  Does it ignore comments?